websocket-client/pom.xml

```diff
@@ -0,0 +1,291 @@
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.wildfly.quickstarts</groupId>
+    <artifactId>wildfly-websocket-client</artifactId>
+    <version>10.0.0-SNAPSHOT</version>
+    <packaging>war</packaging>
+    <name>WildFly Quickstarts: wildfly-websocket-client</name>
+    <description>wildfly-websocket-client: Demonstrates use of a Java (not Javascript) WebSocket client</description>
+ 
+    <url>http://wildfly.org</url>
+    <licenses>
+        <license>
+            <name>Apache License, Version 2.0</name>
+            <distribution>repo</distribution>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.html</url>
+        </license>
+    </licenses>
+
+    <properties>
+        <!-- Explicitly declaring the source encoding eliminates the following message: -->
+        <!-- [WARNING] Using platform encoding (UTF-8 actually) to copy filtered 
+            resources, i.e. build is platform dependent! -->
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <!-- JBoss dependency versions -->
+        <version.wildfly.maven.plugin>1.0.2.Final</version.wildfly.maven.plugin>
+
+        <!-- Define the version of the JBoss BOMs we want to import to specify tested stacks. -->
+        <version.jboss.bom>10.0.0.Beta2</version.jboss.bom>
+        <version.arquillian.container>8.0.0.Final</version.arquillian.container>
+
+        <!-- other plug-in versions -->
+        <version.surefire.plugin>2.16</version.surefire.plugin>
+        <version.war.plugin>2.1.1</version.war.plugin>
+
+        <!-- maven-compiler-plugin -->
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>1.8</maven.compiler.source>
+    </properties>
+
+    <dependencyManagement>
+      <dependencies>
+          <!-- JBoss distributes a complete set of Java EE 7 APIs including a Bill
+              of Materials (BOM). A BOM specifies the versions of a "stack" (or a collection)
+              of artifacts. We use this here so that we always get the correct versions
+              of artifacts. Here we use the jboss-javaee-7.0-with tools stack (you can read this as
+              the JBoss stack of the Java EE 7 APIs, with some extras tools for your project, such
+              as Arquillian for testing) -->
+          <dependency>
+              <groupId>org.wildfly.bom</groupId>
+              <artifactId>jboss-javaee-7.0-wildfly-with-tools</artifactId>
```

Comment:
this changed to <artifactId>wildfly-javaee7-with-tools</artifactId>
after 10.0.0.Beta2